enterprise/internal/authz/gitlab/oauth.go

```diff
@@ -78,6 +85,8 @@ func (p *OAuthProvider) FetchAccount(context.Context, *types.User, []*extsvc.Acc
 // has read access to. The project ID has the same value as it would be
 // used as api.ExternalRepoSpec.ID. The returned list only includes private project IDs.
 //
+// The client used by this method will be in charge of updating the Oauth token if it is expired and retry the request.
```

Comment:
Grammarly says so...

```suggestion
// The client used by this method will be in charge of updating the OAuth token if it has expired and retrying the request.
```